mkdocs.yml

```diff
@@ -95,4 +96,4 @@ plugins:
   - redirects:
       redirect_maps:
         'v6/verification.md': 'v6/operation.md'
-  
+
```

Comment:
It looks like your editor may be cleaning up whitespace automatically, which you should try to avoid/shunt them off ito separate commits in the future (see https://osg-htc.org/technology/software/git-software-development/#avoid-whitespace-noise).